Microsoft.Toolkit.Uwp.UI.Animations/Xaml/AnimationCollection2.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Diagnostics;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Markup;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations.Xaml
+{
+    /// <summary>
+    /// A collection of animations that can be grouped together. This type represents a composite animation
+    /// (such as <see cref="Windows.UI.Xaml.Media.Animation.Storyboard"/>) that can be executed on a given element.
+    /// </summary>
+    [ContentProperty(Name = nameof(Animations))]
+    public sealed class AnimationCollection2 : DependencyObject
+    {
+        /// <summary>
+        /// Raised whenever the current animation is started.
+        /// </summary>
+        public event EventHandler? Started;
+
+        /// <summary>
+        /// Raised whenever the current animation ends.
+        /// </summary>
+        public event EventHandler? Ended;
```

Comment:
Should we name this `Completed` to align with Storyboard event name? (Though they don't have a `Started` event and their `Start` method is called `Begin`...?) Thoughts?